Generator/Sources/needle/PrintDependencyTreeCommand.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  Copyright (c) 2018. Uber Technologies
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import NeedleFramework
+import Utility
+
+/// The default value for waiting timeout in seconds.
+fileprivate let defaultTimeout = 30.0
```

Comment:
Let's move this to the `AbstractCommand` class then. Unless we think this command should potentially have a different timeout value?